### ISSUE TITLE
"window is not defined" error on Next.js 7

### ISSUE THREAD

**Issue by richardvclam** [pqina/react-filepond#28]
### Summary
Importing `react-filepond` into my Next.js 7 application gives the following error:

```
ReferenceError: window is not defined
    at isOperaMini (/Users/richardlam/S2-Projects/React/simple-events/node_modules/filepond/dist/filepond.js:8874:38)
    at /Users/richardlam/S2-Projects/React/simple-events/node_modules/filepond/dist/filepond.js:8901:7
    at /Users/richardlam/S2-Projects/React/simple-events/node_modules/filepond/dist/filepond.js:8914:5
    at isNode (/Users/richardlam/S2-Projects/React/simple-events/node_modules/filepond/dist/filepond.js:10:7)
    at Object.<anonymous> (/Users/richardlam/S2-Projects/React/simple-events/node_modules/filepond/dist/filepond.js:14:3)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/richardlam/S2-Projects/React/simple-events/node_modules/react-filepond/dist/react-filepond.js:24:17)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
```

### How to reproduce

To reproduce, import the following statement into any page on Next and it'll throw the error.

```
import { FilePond, File, registerPlugin } from 'react-filepond';
```

### Additional information

| Environment  | Version
|------------- | -----------
| Filepond | 3.2.3
| React-Filepond | 5.0.0
| React      | 16.6.0
| Nextjs   | 7.0.2
| OS           | MacOS
| Device       | 2017 Macbook Pro
| Browser      | Chrome 70.0.3538.67

**Comment by rikschennink** [comment]
Just published version `3.2.4` of FilePond, please install that package and re-test.

```
npm i filepond@latest --save
```

**Comment by richardvclam** [comment]
It is working beautifully now. Thank you for your efforts!
